corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -602,6 +621,9 @@ cloudCare.AppView = Backbone.View.extend({
 
         return url;
     },
+    getSyncUrl: function () {
+        return "/webforms/player_proxy";
```

Comment:
can we move this to where the other urls are instantiated? https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/cloudcare/templates/cloudcare/cloudcare_home.html#L78